import mysql.connector
from mysql.connector import Error

# try:
conexion = mysql.connector.connect(
    host='localhost',
    port=3306,
    user='root',
    password='12345',
    database='pokeAPI'
)


#     if conexion.is_connected():
#         print('Conexi贸n Exitosa')
#         InfoServer = conexion.get_server_info()
#         print('Informaci贸n del servidor:', InfoServer)
#         cursor = conexion.cursor()
#         cursor.execute("SELECT database();")
#         database = cursor.fetchone()
#         print('Conectado a la base de datos: {}'.format(database))


# except Error as ex:
#     print("Error durante la conexi贸n:", ex)

# finally:
#     if conexion.is_connected():
#         conexion.close()
#         print('La conexi贸n ha finalizado')
